### ISSUE TITLE
Disappearing vaults

### ISSUE THREAD

**Issue by ike5** [padloc/padloc#675]
I've been using Padoc v4.2.0 with my team recently, self-hosted, and several team members let me know that their organization Vaults have disappeared, while others still see theirs.

Here is an image from my point of view of the organization and the vaults inside of it.
<img width="200" alt="Screenshot 2023-01-27 at 6 07 49 PM" src="https://user-images.githubusercontent.com/31159328/215237920-0d3afe31-80ca-47ff-80da-b8eaf2efd0c7.png">

My colleague has been given access to 4 vaults via the Admin group:
<img width="200" alt="Screenshot 2023-01-27 at 6 42 35 PM" src="https://user-images.githubusercontent.com/31159328/215238106-3d3122ee-6def-46c8-b253-5eff4f5feb85.png">

Here's an image from my colleague's point of view where the organization isn't showing their vaults (but it shows the spaces for the 4 vaults they should have access to).
<img width="200" alt="Screenshot 2023-01-27 at 6 07 38 PM" src="https://user-images.githubusercontent.com/31159328/215238039-bfdb96d3-b2fe-443c-ac4e-f72c96b835df.png">

Users are originally able to see their vaults after being invited to joint the organization, but they lose access to it.

**Comment by MaKleSoft** [comment]
Hey @ike5, thanks for reporting this! Can you you check if there are any error messages in the browser console for those team members that have lost access?

**Comment by ike5** [comment]
@MaKleSoft Sure. I'll try and see if I can get a screenshot by Monday. Thanks for looking into this.

**Comment by ike5** [comment]
Here are screenshots of the browser console from two different people.

<img width="1500" alt="Screenshot 2023-01-30 at 2 10 36 PM" src="https://user-images.githubusercontent.com/31159328/215607364-1f0959cb-ccde-4728-829e-435dc42b0e39.png">
<img width="1433" alt="Screenshot 2023-01-30 at 2 09 14 PM" src="https://user-images.githubusercontent.com/31159328/215607440-ec75a9b8-6244-4d6a-91f5-64713b223580.png">

**Comment by MaKleSoft** [comment]
Looks like those two members have been suspended. Is it possible they reset their master password after joining? See also https://docs.padloc.app/manual/orgs/#suspending-and-un-suspending-members

**Comment by ike5** [comment]
> Looks like those two members have been suspended. Is it possible they reset their master password after joining? See also https://docs.padloc.app/manual/orgs/#suspending-and-un-suspending-members

7 other members logged in and found themselves in similar situations without being able to see the organization--but not all at once. Each of their consoles was showing similar suspensions messages. I know at least 2 reset their passwords and were suspended that way, so it probably explains that. 

Today I just rotated the cryptographic keys for the first time and it looks like folks are able to use their accounts again (after confirming their invites). Sorry for this bug being unclear, since it was resolved by rotating the keys.
